LICENSE

```diff
@@ -0,0 +1,23 @@
+MIT License
```

Comment:
nit: I suspect if we remove this line then GitHub might automatically pick up that this is the MIT license.

Reply:
That's an interesting one actually, as I used the Github utility to generate this file..

Reply:
huh, I wonder if it's something else that means it's not showing the license banner thingy then.

Reply:
It won't show the banner until the licence is on master I suspect. See a very similar file with that heading on this repo: https://github.com/j5api/j5/

Reply:
It won't show the banner until the licence is on master I suspect.